.travis.yml

```diff
@@ -9,6 +9,9 @@ env:
   global:
     - setup=basic
 
+git:
+  depth: 1
```

Comment:
<strike>Because there is no reason cloning whole history for most of Travis jobs.<strike>

Have you read Travis note on depth 1?

>Please note that if you use a depth of 1 and have a queue of jobs, Travis CI won’t build commits that are in the queue when you push a new commit.